src/components/Footer/index.tsx

```diff
@@ -85,34 +188,39 @@ export const Footer: React.FC<FooterType> = props => {
 
             <Cell cols={5} colsM={6} colsS={8}>
               <p className={`${classes.colHeader} ${classes.thirdColumn}`}>Stay connected</p>
-
               <div>
-                <form
-                  method="POST"
-                  action="https://payloadcms.us18.list-manage.com/subscribe/post?u=f43c9eb62d4ce02e552a1fa9f&amp;id=e11798f237"
-                  onSubmit={() => analyticsEvent('newsletter')}
-                >
+                {error && <div>{`${error.status || '500'}: ${error.message || ''}`}</div>}
```

Comment:
Is there a need to expose this to the user?

Reply:
We are doing it with our regular forms as well. Not entirely necessary I suppose. What do you think?